partner_customer_supplier/tests/test_res_partner.py

```diff
@@ -0,0 +1,81 @@
+from odoo.tests import common, tagged
+
+
+@tagged("res_partner")
+class TestResPartner(common.TransactionCase):
+    def setUp(self):
+        super().setUp()
+        self.partner = self.env["res.partner"].create({"name": "Microsoft Corporation"})
+        self.partner_2 = self.env["res.partner"].create({"name": "Apple Inc."})
```

Comment:
you want to see the world burn! 🔥 hahaha

```suggestion
        self.partner = self.env["res.partner"].create({"name": "Microsoft Corporation"})
        self.partner_2 = self.env["res.partner"].create({"name": "Apple Inc."})
```